dataland-frontend/src/assets/scss/variables.scss

```diff
@@ -15,7 +15,7 @@ $brown: #292219;
 $brown-light: #5A4F36;
 $brown-lighter: #958D7C;
 $gray: #6c757d;
-$gray-light: #d2d2d2;
+$gray-light: #e0e0e0;
```

Comment:
Why? We do not use it in this branch.

Reply:
Maybe it is used somewhere in the table?
Maybe @MarcinStanczyk7 knows why this was changed.

Reply:
because it looked a little brighter on figma

Reply:
![image](https://github.com/d-fine/Dataland/assets/96523011/a92b0711-98cf-40a8-a7ce-8175f332b831)
ilusion